modules/cms/models/ThemeData.php

```diff
@@ -113,19 +113,24 @@ public function afterFetch()
     {
         $data = (array) $this->data + $this->getDefaultValues();
 
-        /*
-         * Repeater form fields store arrays and must be jsonable.
-         */
         foreach ($this->getFormFields() as $id => $field) {
             if (!isset($field['type'])) {
                 continue;
             }
 
+            /*
+             * Repeater form fields store arrays and must be jsonable.
+             */
             if ($field['type'] === 'repeater') {
                 $this->jsonable[] = $id;
             }
             elseif ($field['type'] === 'fileupload') {
-                $this->attachOne[$id] = File::class;
+                if (array_get($field, 'multiple', false)) {
+                    $this->attachMany[$id] = File::class;
+                }
+                else {
```

Comment:
Can you move the else statement here and on the above line to the same line as the closing curly brace?

Reply:
Done.